src/main/java/net/sf/jsqlparser/schema/Column.java

```diff
@@ -43,23 +43,28 @@ public Column(String columnName) {
      * Retrieve the information regarding the {@code Table} this {@code Column} does
      * belong to, if any can be inferred.
      * <p>
-     * The inference is based only on local information, and not on the whole SQL command.
-     * For example, consider the following query:
-     * <blockquote><pre>
+     * The inference is based only on local information, and not on the whole SQL
```

Comment:
formatting got messed up her, does the javadoc still render correctly?

Reply:
https://github.com/JSQLParser/JSqlParser/pull/1014/commits/db3b4ae4e6f83f1a342c9ce0cacadc8618b5c16e